Makefile

```diff
@@ -73,7 +73,7 @@ rpm-release: srpm-release
 	mock -r $(MOCK_CONFIG) --resultdir BUILD/RPM -D "rel_build 1" --rebuild BUILD/SRPM/python-aexpect-$(VERSION)-*.src.rpm
 
 check: clean
-	inspekt checkall --disable-lint W1618,R0205
+	inspekt checkall --disable-lint W1618,R0205,W0707,R1725
```

Comment:
Is there a reason to add these? I tried running it with W0707 and R1725 enabled and it passed. I'd really prefer addressing the actual places only.

Reply:
See: https://travis-ci.org/github/clebergnu/aexpect/jobs/729403637#L263 and https://travis-ci.org/github/clebergnu/aexpect/jobs/729403637#L274

Reply:
I see. If we decide to drop py2 we should also address all these (especially the W0707) as they require py3. If we kept py2 skipping these is fine.

Reply:
> I see. If we decide to drop py2 we should also address all these (especially the W0707) as they require py3. If we kept py2 skipping these is fine.

Are you volunteering? :)